Assets/Scripts/Player/PlayerCamera.cs

```diff
@@ -3,50 +3,73 @@
 
 public class PlayerCamera : ActorComponent 
 {
-	public Camera cam;
+	ActorPhysics actorPhysics = null; // Need this so we can change the camera in different states
```

Comment:
replace this with calls to actor.physics. You may have to pull in updates from master first if that's not in yet on this branch.